### ISSUE TITLE
portkey sometimes errors on buffer open

### ISSUE THREAD

**Issue by hjdivad** [dsawardekar/portkey#7]
In my config I get errors when I `sudo vim <file>` although not when i merely `sudo vim` and not when I run vim normally.

Excerpt from logs:

```
Executing BufRead Auto commands for "*"
autocommand call s:plugin.on_buffer_open(expand('<afile>'))

Error detected while processing function <SNR>54_s:Plugin_on_buffer_open:
line   12:
```

script 54 is `portkey/plugin/portkey.vim`

Any ideas?  Anything more I can provide to help debug?

**Comment by dsawardekar** [comment]
I tried this on Ubuntu. Seems fine with Vim 7.4, 7.3p905 and master. I'm guessing this may be related to a specific version of Vim. What version are you on?

You could try setting `verbosefile` to get a more detailed log of what's hapenning.

**Comment by hjdivad** [comment]
Ah yes, sorry.  I keep forgetting that sudo will drop me into `/usr/bin/vim` which on my box is 7.3.  I've fixed this so now it's all 7.4 and I don't get this error any longer.
